deploy/kubernetes/base/poddisruptionbudget-controller.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Add this new file to `kustomization.yaml`. Is there a way be can also automate this?

Reply:
good catch